mitodl/open-discussions
# Replace 'channel' with 'c' in URLs, redirect old URLs to new ones

#### What are the relevant tickets?
Fixes #867

#### What's this PR do?
Replaces `channel/` with `c/` in URLs.

#### How should this be manually tested?
- Navigate to various channel, post, comment URL's.  They should all start with `c/` instead of `channel/`.   Try replacing `c/` with `channel/`, you should be redirected to the `c` URL.
- Click 'View reported posts & comments' for a channel you have moderation access to.  The link should include `moderation/c/` instead of `moderation/channel` and should work properly.
- Channel management links should work with a `c` instead of `channel`, for example `http://od.odl.local:8063/manage/c/edit/<your_channel>/basic`
